packages/ariakit/src/popover/__examples__/popover-portal/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button } from "ariakit/button";
+import { Popover, usePopoverState } from "ariakit/popover";
+import "../popover/style.css";
```

Comment:
[This doesn't work](https://ariakit-git-fork-georgekaran-fix-portal-focus-loop-ariakit.vercel.app/examples/popover-portal). You have to create a `style.css` file inside this folder and `@import url("../popover/style.css")` from there.
```suggestion
import "./style.css";
```